package.json

```diff
@@ -16,7 +16,8 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "client": "cd client && npm start",
     "server": "nodemon server.js",
-    "start": "node server.js"
+    "start": "node server.js",
+    "api": "cd client && npx json-server --watch db.json --port 5000"
```

Comment:
I would suggest you install json-server and implement it into your server.js
https://www.npmjs.com/package/json-server#module